enums/productCode.ts

```diff
@@ -2,6 +2,8 @@ export enum ProductCode {
   BUSINESS = 'BUSINESS',
   CSO = 'CSO',
   PPR = 'PPR',
+  MHR = 'MHR',
+  PPR_MHR = 'PPR_MHR',
```

Comment:
o interesting, there is a code explicitly for having both product subscriptions?

Reply:
I wanted to keep the same way we display tiles on the page, and to display one 'My Asset' tile (instead of MHR tile and PPR tile) I added this product. Based on this product id we get all the content/links from the user-product-helper.

Reply:
If it is a code unique to this UI perhaps leave a comment describing as such. I imagine these codes correlate to a product list that originates in Auth, so if we have something unique here we should highlight that.

Reply:
Refactored and removed this products alltogether

Reply:
Awesome!